### ISSUE TITLE
SST39SF040A

### ISSUE THREAD

**Issue by lindoran** [slu4coder/The-Minimal-64-Home-Computer#5]
In going over the BOM (I am trying to figure out sourcing some of the parts leading up to the PCB release) is the 
SST39SF040 going to be ok?

https://www.mouser.com/ProductDetail/Microchip-Technology-Atmel/SST39SF040-70-4C-PHE?qs=YClUa%252B2dcx1pgizrqJ6nyQ%3D%3D

I am thinking possibly yes. -- I am having a very difficult time finding this one from even a sketchy source 

Thanks for your time.

Dave

**Comment by denjhang** [comment]
> In going over the BOM (I am trying to figure out sourcing some of the parts leading up to the PCB release) is the SST39SF040 going to be ok?
> 
> https://www.mouser.com/ProductDetail/Microchip-Technology-Atmel/SST39SF040-70-4C-PHE?qs=YClUa%252B2dcx1pgizrqJ6nyQ%3D%3D
> 
> I am thinking possibly yes. -- I am having a very difficult time finding this one from even a sketchy source
> 
> Thanks for your time.
> 
> Dave

I don't think this is a problem because I already have several hundred SST39SF040, this FlashROM is arguably one of the most commonly used memory chips in the retro computer world. You can try AliExpress (beware of fakes).

**Comment by lindoran** [comment]
Thanks :)
